kubernetes-azure-python/__main__.py

```diff
@@ -0,0 +1,122 @@
+"""An Azure RM Python Pulumi program"""
+
+import base64
+import pulumi
+from pulumi_azure_native import resources
+from pulumi_azure_native import network
+from pulumi_azure_native import containerservice
+
+# Get some values from the Pulumi configuration (or use defaults)
+# First pull provider-namespaced configuration values
+azure_cfg = pulumi.Config("azure-native")
+azureLocation = azure_cfg.require("location")
+# Second get project-namespaced configuration values
+proj_cfg = pulumi.Config()
+num_worker_nodes = proj_cfg.get_float("numWorkerNodes", 3)
+k8s_version = proj_cfg.get("kubernetesVersion", "1.24.3")
+prefix_for_dns = proj_cfg.get("prefixForDns", "pulumi")
+node_vm_size = proj_cfg.get("nodeVmSize", "Standard_DS2_v2")
+mgmt_group_id = proj_cfg.require("mgmtGroupId")
+ssh_pub_key = proj_cfg.require("sshPubKey")
+
+# Create an Azure Resource Group
+resource_group = resources.ResourceGroup("resource_group",
+    location=azureLocation,
+    resource_group_name="aks-rg"
+)
+
+# Create an Azure Virtual Network
+virtual_network = network.VirtualNetwork("virtual_network",
+    address_space=network.AddressSpaceArgs(
+        address_prefixes=["10.0.0.0/16"],
+    ),
+    location=azureLocation,
+    resource_group_name=resource_group.name,
+    virtual_network_name="aks-vnet"
+)
+
+# Create three subnets in the virtual network
+subnet1 = network.Subnet("subnet-1",
+    address_prefix="10.0.0.0/22",
+    resource_group_name=resource_group.name,
+    subnet_name="aks-subnet-1",
+    virtual_network_name=virtual_network.name
+)
```

Comment:
```suggestion
subnet1 = network.Subnet(
    "subnet-1",
    address_prefix="10.0.0.0/22",
    resource_group_name=resource_group.name,
    subnet_name="aks-subnet-1",
    virtual_network_name=virtual_network.name
)
```

Reply:
Addressed in 5b8b6ce (should be correct now)